src/main/scala/akka/persistence/jdbc/query/JournalSequenceActor.scala

```diff
@@ -1,3 +1,8 @@
+/*
+ * Copyright (C) 2014 - 2019 Dennis Vriend <https://github.com/dnvriend>
+ * Copyright (C) 2019 - 2019 Lightbend Inc. <https://www.lightbend.com>
+ */
```

Comment:
I think this is a good example. New files will also get `Copyright (C) 2014 - 2019 Dennis Vriend <https://github.com/dnvriend>`

Reply:
All files get the same header.